### ISSUE TITLE
Don't use prefix when use hosts

### ISSUE THREAD

**Issue by andrefaria** [enriclluelles/route_translator#109]
I really need to not hide the prefixes using different hosts:
- http://abc.io/brands
- http://abc.com.br/marcas

instead of 
- http://abc.io/en/brands
- http://abc.com.br/pt-br/marcas

I'm aware of the risk of conflict routes, but this is really necessary in my case, I can't afford to have the prefixes. 

Do you guys think it could be possible to make this work?

RouteTranslator.config do |config|
        config.host_locales = { '*.io' => :en, '*.com.br'    => :pt_BR }        
        config.hide_locale!
end

**Comment by tagliala** [comment]
This doesn't work with the default implementation of `host_locales`. I'm using a custom solution

**Comment by wagnerads** [comment]
Do you have a fork or something similar? I need the same thing...

**Comment by tagliala** [comment]
Just use `hide_locale = true` and make something like this in the application controller

```rb
  skip_around_action :set_locale_from_url
  around_action :set_site_locale

  # ...

  def set_site_locale(&block)
    I18n.with_locale DOMAIN_LANGUAGE[request.host], &block
  end
```

DOMAIN_LANGUAGE is a hash like this

```
  # Map domains to languages
  DOMAIN_LANGUAGE = {
    'it.lvh.me' => :it,
    'en.lvh.me' => :en,
    'de.lvh.me' => :de
  }
```

**Comment by tagliala** [comment]
Is there someone available to work on this?

**Comment by tagliala** [comment]
@andrefaria could you please give a try to 4.2.2 ?

**Comment by andrefaria** [comment]
@tagliala version 4.2.2 is ok! =) 

Thanks!

**Comment by tagliala** [comment]
perfect, closed via #119
